README.md

```diff
@@ -16,14 +13,16 @@ You can do this under the `Files Changed` tab of Github's PR page. Then click th
 2. The `merge` script depends on [json](http://trentm.com/json/). You can install it using `sudo npm install -g json`.
 NPM depends on [Node.js](https://nodejs.org).
 
-3. You also need to exchanged ssh keys with github. This requires two steps:
+3. You also need to exchange ssh keys with github. This requires two steps:
   * Generate ssh [keys](https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/)
   * Exchage public key with [github](https://help.github.com/articles/adding-a-new-ssh-key-to-your-github-account/)
 
-4. Now from wihtin the `devtools` directory execute merge as follows:
+4. In addition ensure the PR has a version label assigned to it.
+
+5. Now run the `merge` script as follows:
 
 ```
-→ bin/merge <github-repo-name> <PR-number>
+→ merge <github-repo-name> <PR-number>
```

Comment:
Update might be required depending on the answer to the interface question below.